README.md

```diff
@@ -83,26 +83,48 @@ The same process is followed for the asset identifier.
 The identifier is decomposed into its namespace, reference, and optional ID.
 Then, the resolver for the specified asset namespace is used to resolve the asset reference and ID.
 
+### JSON-LD Context
+
+The snippet below shows the context that the process of AssetDID resolution returns.
+
+```json
+{
+  "@protected": true,
+  "id": "@id",
+  "asset-did-spec": "https://github.com/KILTprotocol/spec-asset-did/blob/main/README.md",
```

Comment:
Did you check what the predicates expand to? Not sure, but you might need to add a `#` to make sure the result is a valid URL